protocols/v2/codec-sv2/Cargo.toml

```diff
@@ -16,9 +16,10 @@ const_sv2 = { version = "2.0.0", path = "../../../protocols/v2/const-sv2"}
 buffer_sv2 = { version = "1.0.0", path = "../../../utils/buffer"}
 tracing = { version = "0.1"}
 
-
+[dev-dependencies]
+key-utils = { version = "^1.0.0", path = "../../../utils/key-utils" }
 
 [features]
 with_serde = ["binary_sv2/with_serde", "serde", "framing_sv2/with_serde", "buffer_sv2/with_serde"]
 with_buffer_pool = ["framing_sv2/with_buffer_pool"]
-no_std = []
```

Comment:
is this EOF line removal intended?

Reply:
Again, I do not know what the GitHub UI is showing this was removed. This line is still present in the toml.